eng/service.proj

```diff
@@ -8,7 +8,7 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <ExcludeMgmtLib Include="..\sdk\*\*.Management.*\**\*.csproj;..\sdk\*mgmt*\**\*.csproj;" />
+    <ExcludeMgmtLib Include="..\sdk\*\Microsoft.*.Management.*\**\*.csproj;..\sdk\*mgmt*\**\*.csproj;" />
```

Comment:
@weshaggard I'm not sure about this change, feels like we shouldn't be treating Track 2 management libraries differently from data-plane but it's your call.

Reply:
They are coming, I've pulled this change out of https://github.com/Azure/azure-sdk-for-net/pull/10485 and management team is actively trying new generator on management swaggers.

Reply:
OK I'm good with this for now.